tests/dummy/app/templates/demos/sl-tab-panel.hbs

```diff
@@ -21,8 +26,13 @@
     <div class="col-lg-6">
         <h6>Rendered Component</h6>
         {{#sl-tab-panel}}
-            {{sl-tab-pane label="One" name="one" templateName="demos/tabs/one"}}
-            {{sl-tab-pane label="Two" name="two" templateName="demos/tabs/two"}}
+            {{#sl-tab-pane label="One" name="one"}}
+                {{render "demos.tabs.one"}}
```

Comment:
The 11th bullet point in the http://emberjs.com/blog/2015/05/24/another-two-oh-status-update.html#toc_removals list states that the `{{render}}` helper is being removed in 2.x, being superseded in most cases by components